node/cli/Cargo.toml

```diff
@@ -18,19 +24,18 @@ sc-telemetry = { git = "https://github.com/purestake/substrate", branch = "moonb
 frame-benchmarking-cli = { git = "https://github.com/purestake/substrate", branch = "moonbeam-polkadot-v0.9.12" }
 try-runtime-cli = { git = "https://github.com/purestake/substrate", branch = "moonbeam-polkadot-v0.9.12", optional = true }
 
+# Cumulus
 cumulus-client-cli = { git = "https://github.com/purestake/cumulus", branch = "moonbeam-polkadot-v0.9.12" }
 cumulus-client-service = { git = "https://github.com/purestake/cumulus", branch = "moonbeam-polkadot-v0.9.12" }
 cumulus-primitives-core = { git = "https://github.com/purestake/cumulus", branch = "moonbeam-polkadot-v0.9.12" }
 nimbus-primitives = { git = "https://github.com/purestake/nimbus", branch = "moonbeam-polkadot-v0.9.12" }
```

Comment:
Not really part of cumulus anymore. There aren't that many crates there, so I don't know if it warrants its own section or not. But if not does it make more sense under Moonbeam or just in the general section?